<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1797" started-at="2019-08-11T12:07:11Z" finished-at="2019-08-11T12:07:13Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1797" started-at="2019-08-11T12:07:11Z" finished-at="2019-08-11T12:07:13Z">
      <class name="com.Vcentry.tnstc.testcases.RegPageTest">
        <test-method status="SKIP" signature="validateRegpage(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Vcentry.tnstc.testcases.RegPageTest@43a25848]" name="validateRegpage" duration-ms="0" started-at="2019-08-11T17:37:12Z" data-provider="RegPageTest" finished-at="2019-08-11T17:37:12Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.io.FileNotFoundException: C:\Users\shanwik\workspace2\DDDTest\src\test\resources\com\Vcentry\tnstc\inputs\inputcontroller.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: C:\Users\shanwik\workspace2\DDDTest\src\test\resources\com\Vcentry\tnstc\inputs\inputcontroller.xlsx (The system cannot find the path specified)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:163)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
Caused by: java.io.FileNotFoundException: C:\Users\shanwik\workspace2\DDDTest\src\test\resources\com\Vcentry\tnstc\inputs\inputcontroller.xlsx (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at com.Vcentry.tnstc.inputreader.inputReader.selectSingleDataOrMulitiData(inputReader.java:45)
	at com.Vcentry.tnstc.testcases.RegPageTest.getData(RegPageTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateRegpage -->
      </class> <!-- com.Vcentry.tnstc.testcases.RegPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
